d2l-fetch-simple-cache.js

```diff
@@ -1,4 +0,0 @@
-import simpleCache from './src/index.js';
-
-window.d2lfetch = window.d2lfetch || {};
-window.d2lfetch.simpleCache = simpleCache;
```

Comment:
This file wasn't actually referenced anywhere and I see no uses of this global on the `window`, so deleted it.